src/Moq/Invocation.cs

```diff
@@ -69,16 +69,25 @@ public MethodInfo MethodImplementation
 
 		public Type ProxyType => this.proxyType;
 
+		/// <inheritdoc cref="IInvocation.ReturnValue"/>
```

Comment:
I don't know if this is needed/supported and I have not checked whether it is working or not.